documentation/translations/translating.rst

```diff
@@ -283,6 +283,30 @@ Some useful resources:
 Translation FAQ
 ===============
 
+.. _docs-build-translation:
+
+How do I build a docs translation?
+----------------------------------
+
+To build a documentation translation, you need to have Python installed and a
+local copy of the :github:`CPython repository <python/cpython>` and
+translation repository (see table above). The PO files must be placed
+in a :samp:`locales/{LANG}/LC_MESSAGES/` (replacing :samp:`{LANG}` with the translation's
+language code) folder inside the :file:`Doc/` directory of the CPython repository.
+
+You can then build with :ref:`make <doc-build-make>` by adding
+a ``SPHINXOPTS="-D language=LANG"`` variable before the target
+or by using :ref:`Sphinx directly <doc-build-sphinx>` and adding a
+``-D language=LANG`` option. For example:
+
+.. code-block:: bash
+
+    # build the html format of the Polish translation using make
+    make SPHINXOPTS="-D language=pl" html
```

Comment:
```suggestion
    make SPHINXOPTS="-D language=pl" html
    
```

Reply:
I think that adding a space will indicate better that these are two different examples.